tests/ui/legacy_integral_constants.stderr

```diff
@@ -0,0 +1,72 @@
+error: usage of a legacy integral constant
+  --> $DIR/legacy_integral_constants.rs:10:5
+   |
+LL |     std::f32::EPSILON;
+   |     ^^^^^^^^^^^^^^^^^ help: try: `f32::EPSILON`
+   |
+   = note: `-D clippy::legacy-integral-constants` implied by `-D warnings`
```

Comment:
The two successive colons in `` help: try: `f32::epsilon` `` looks off to me. I'd also like to see the help message mention "associated constant" somewhere. Something like this:

```
error: usage of a legacy integral constant
  --> $DIR/legacy_integral_constants.rs:10:5
   |
LL |     std::f32::EPSILON;
   |     ^^^^^^^^^^^^^^^^^ help: use the associated constant `f32::EPSILON` instead
   |
```

Reply:
That's how inline suggestions look, no real way around it whilst still having it applied unfortunately

Reply:
Alright, in that case the following would be fine

    help: use the associated constant instead: `f32::EPSILON`